### ISSUE TITLE
Treating the internal errors in the IDE

### ISSUE THREAD

**Issue by viper-admin** [viperproject/viper-ide#83]
> Created by **@aterga** on 2017-03-20 10:20
> Last updated on 2017-03-20 14:35

Sometimes the backends fail. If the type of failure is not one we have encountered in the extension, we currently categorize it as "Internal Error". Internal errors cause a pop-up mesage in Code. 

Because some very strateforward usage scenarios cause such errors we have to minimize the overhead of recovering from internal erros. For example, one can start manually renaming a type parameter in a domain definition and it's axioms, which, at an intermediate point of time, may introduce invalid quantifier triggers, causing internal errors. I think that in this case a pop-up is not at all necessary. 

TL;DR I propose that we do not show pop-ups for internal errors.
